provisioning/baremetal_secrets.go

```diff
@@ -159,10 +157,6 @@ func EnsureAllSecrets(info *ProvisioningInfo) (bool, error) {
 	if err := createIronicSecret(info, ironicSecretName, ironicUsername, "ironic"); err != nil {
 		return false, errors.Wrap(err, "failed to create Ironic password")
 	}
-	// Create a Secret for the Ironic Inspector Password
-	if err := createIronicSecret(info, inspectorSecretName, inspectorUsername, "inspector"); err != nil {
```

Comment:
I wonder if we should delete the existing secret.

Reply:
Good point, have deleted it at the end of this function now